src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -438,5 +438,20 @@ internal override IEnumerable<CSharpAttributeData> GetCustomAttributesToEmit(PEM
         internal sealed override bool IsRecord => _underlyingType.IsRecord;
         internal sealed override bool IsRecordStruct => _underlyingType.IsRecordStruct;
         internal sealed override bool HasPossibleWellKnownCloneMethod() => _underlyingType.HasPossibleWellKnownCloneMethod();
+
+#nullable enable
+        internal sealed override bool IsExtension => _underlyingType.IsExtension;
+
+        internal sealed override TypeSymbol? ExtensionUnderlyingTypeNoUseSiteDiagnostics
+            => _unbound ? null : Map.SubstituteType(OriginalDefinition.ExtensionUnderlyingTypeNoUseSiteDiagnostics).Type;
+
+        internal sealed override ImmutableArray<NamedTypeSymbol> BaseExtensionsNoUseSiteDiagnostics
+            => _unbound ? ImmutableArray<NamedTypeSymbol>.Empty : Map.SubstituteNamedTypes(OriginalDefinition.BaseExtensionsNoUseSiteDiagnostics);
+
+        internal sealed override TypeSymbol? GetDeclaredExtensionUnderlyingType()
+            => throw new InvalidOperationException("PROTOTYPE"); // PROTOTYPE
```

Comment:
>throw new InvalidOperationException("PROTOTYPE"); // PROTOTYPE [](http://example.com/codeflow?start=15&length=62)

We are probably going to hit this with lookup. It is fine to not deal with this for now, but let's make sure we have good coverage with generic extensions